internal/config/image/builder.go

```diff
@@ -100,3 +110,32 @@ func WithMounts(mounts []specs.Mount) Option {
 		return nil
 	}
 }
+
+// WithPrivileged sets whether an image is privileged or not.
+func WithPrivileged(isPrivileged bool) Option {
+	return func(b *builder) error {
+		b.isPrivileged = isPrivileged
+		return nil
+	}
+}
+
+// WithAdditionalVisibleDevicesEnvVars sets the visible devices environment variables.
+// For each envvar passed:
+//   - Split on commas
+//   - Trim spaces and ignore empty strings
+//   - Concatenate all of this
+func WithAdditionalVisibleDevicesEnvVars(visibleDevicesEnvVars ...string) Option {
+	return func(b *builder) error {
+		var result []string
+		for _, v := range visibleDevicesEnvVars {
+			for _, c := range strings.Split(v, ",") {
+				trimmed := strings.TrimSpace(c)
+				if trimmed != "" {
+					result = append(result, trimmed)
+				}
+			}
+		}
+		b.additionalVisibleDevicesEnvVars = result
+		return nil
+	}
```

Comment:
```suggestion
	return func(b *builder) error {
		b.additionalVisibleDevicesEnvVars = visibleDevicesEnvVars
		return nil
	}
```
(also note that the argument name hasn't been changed to matche the member / function name).